test/output/slim/document.html

```diff
@@ -0,0 +1,2194 @@
+<!-- .title -->
```

Comment:
Document examples should be scoped (using `:include:` option), look at https://github.com/jirutka/asciidoctor-html5s/blob/master/test/examples/html5/document.html for inspiration.